secp256k1-sys/src/lib.rs

```diff
@@ -125,6 +139,55 @@ impl Default for Signature {
     }
 }
 
+#[repr(C)]
+pub struct XOnlyPublicKey([c_uchar; 64]);
+impl_array_newtype!(XOnlyPublicKey, c_uchar, 64);
+impl_raw_debug!(XOnlyPublicKey);
+
+impl XOnlyPublicKey {
+    /// Create a new (zeroed) x-only public key usable for the FFI interface
+    pub fn new() -> XOnlyPublicKey { XOnlyPublicKey([0; 64]) }
```

Comment:
We shouldn't have this function, as it creates an invalid public key which will trigger assertation failures in upstream.

Reply:
I'll fix this in a followup commit/PR. It's a bit subtle because we do need (temporarily) uninitialized values to use the FFI interface.